support/doc/api/openapi.yaml

```diff
@@ -3263,6 +3263,62 @@ paths:
         '204':
           description: successful operation
 
+  '/video-channel-syncs/':
+    post:
+      summary: Create a synchronization for a video channels
+      operationId: addVideoChannelSync
+      security:
+        - OAuth2: []
+      tags:
+        - Video Channels Synchronizations
```

Comment:
`Channels Sync` (we need to be concise because it is displayed on the left menu in https://docs.joinpeertube.org/api-rest-reference.html

You'll also have to add this tag to the list of available tags in https://github.com/Chocobozzz/PeerTube/blob/develop/support/doc/api/openapi.yaml#L154 and in https://github.com/Chocobozzz/PeerTube/blob/develop/support/doc/api/openapi.yaml#L300